.gitignore

```diff
@@ -1 +1,26 @@
-node_modules
\ No newline at end of file
+node_modules
+**/aws.json
```

Comment:
Stuff which is related to the buidler can actually stay in a .gitignore in the buidler folder so things can be kept in context to the package

Im not bothered - but I guess its useful info for you to know if you use workspaces in future ^_^